boards/FSM/Src/Io/Io_Brake.c

```diff
@@ -3,8 +3,8 @@
 #include "Io_Adc.h"
 
 // TODO set these values
-#define BRAKE_PEDAL_MIN_VOLTAGE (0.5f)
-#define BRAKE_PEDAL_MAX_VOLTAGE (4.5f)
+#define BRAKE_PEDAL_MIN_VOLTAGE (2.08f)
+#define BRAKE_PEDAL_MAX_VOLTAGE (2.310f)
```

Comment:
Note: these numbers will need to be re-tuned when/if we ever fix the potentiometer to stop slipping

Reply:
sounds good